src/main/webapp/user-private-comments.jsp

```diff
@@ -14,27 +13,53 @@
 %>
 
 <%--HTML--%>
-<ul>
-  <%if (currUser) {%>
-    <h1 id="pc-header">Your Private Task Comments:</h1>
-    <%for (TaskData task : tasks) {%>
-      <li class="task-with-pc">
-        <button class="inline deep-button" type="button" onclick="location.href='task?taskID=<%=task.getTaskID()%>'">
-          <%=task.getName()%>
-        </button>
-        <%request.setAttribute("task", task);%>
-        <%request.setAttribute("clickable", false);%>
-        <jsp:include page="task-status-checkmark.jsp"/>
-        <p class="inline"><%=task.getDescription()%></p>
-        <form action="/user-profile" method="POST">
-          <input type="hidden" name="taskID" value="<%=task.getTaskID()%>">
-          <br>
-          <textarea style="width:300px; height:100px;" type="text" name="message-<%=task.getTaskID()%>" rows="6" cols="50"><%=(privateCommentsMap.get(task.getTaskID())).getMessage().trim()%></textarea>
-          <br>
-          <button class="deep-button" style="font-size: 12px;" type="submit" class="deep-button">Update Comment</button>
-        </form>
-        <br><br>
-      </li>
+<head>
+  <script src="https://cdn.jsdelivr.net/npm/@editorjs/editorjs@latest"></script>
+  <script src="https://cdn.jsdelivr.net/npm/@editorjs/header@latest"></script>
+</head>   
+  <ul>
+    <%if (currUser) {%>
+      <h1 id="pc-header">Your Private Task Comments:</h1>
+      <%for (TaskData task : tasks) {%>
```

Comment:
Since you have your "init()" and "save()" methods in a loop, this will create multiple copies of it.  This is ok in Javascript (other languages would complain), but it'll only keep the last version (the one for the last task).

You also have your <div id="box"> inside the loop, which will be a problem for the editorjs since it can't detect which box is the correct box.

What you can do instead:
-  Move init() and save() into the js file, but have the methods be flexible enough to work for the different tasks (maybe pass in a privateCommentId so you know which box to build).
- This will also mean you'll have multiple "editor" objects (one per comment), so you'll need to store that in an array